qgepplugin/tools/qgepmaptooladdfeature.py

```diff
@@ -65,7 +65,8 @@ def __init__(self, map_canvas: QgsMapCanvas, geometry_type):
     def addPoint3D(self, point):
         assert type(point) == QgsPoint
         QgsRubberBand.addPoint(self, QgsPointXY(point.x(), point.y()))
-        self.points.append(QgsPoint(point))
+        new_point = QgsPoint(point.x(), point.y(), point.z(), point.m(), point.wkbType())
```

Comment:
@olivierdalang can you leave a comment here, why we use this workaround. This will

1. make sure we don't "improve" the code as long as there is no upstream fix
2. make sure we can improve the code with confidence (including minimum QGIS version) once there is an upstream fix.

Reply:
```suggestion
        # Workaround crash with QGIS 3.10.2 (https://github.com/qgis/QGIS/issues/34557)
        new_point = QgsPoint(point.x(), point.y(), point.z(), point.m(), point.wkbType())
```